cmd/kev/cmd/dev.go

```diff
@@ -83,12 +80,8 @@ func init() {
 		"Override default Kubernetes manifests output directory. Default: k8s/<env>",
 	)
 
-	flags.StringSliceP(
-		"environment",
-		"e",
-		[]string{"dev"},
-		"Specify an environment",
-	)
+	flags.StringSlice("environment", []string{}, "")
```

Comment:
I guess we can remove it completely from `dev` command now as render was the only thing potentially taking that into account? `dev` will simply rebuild all tracked environments which simplifies the logic a little bit.

Reply:
@marcinc Can't because we use the `runRenderCmd(cmd, args)` directly. Which expects `environment` flag.

Reply:
ah, makes sense.